.claude/agents/software-architect.md

```
@@ -0,0 +1,117 @@
+---
+name: software-architect
+description: Expert software architect specializing in Python semantics and Rust project design. Use when hitting architectural walls or needing deep technical guidance.
+tools: Read, Glob, Grep, Bash, WebFetch, mcp__filesystem__*, mcp__github__*, mcp__lsmcp__*
+model: opus
+---
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)